.packit.yaml

```diff
@@ -10,6 +10,8 @@ files_to_sync:
       dest: plans/
     - src: .fmf/
       dest: .fmf/
+    - src: tests/
```

Comment:
We haven't had to do that before. What's changed?

Reply:
Why didn't we have to do that? I think it was missing by mistake. Downstream CI tried to execute the tmt plan but could not find the test.

Reply:
Does not look as it was not found: https://artifacts.dev.testing-farm.io/78e9d3cd-7980-472a-920e-1e7b4d6454ae/

Reply:
Packit itself does not sync anything else as well: https://github.com/packit/packit/blob/8d661ceb831aacbf6ef39d186ec8ef850e800dad/.packit.yaml#L6

Reply:
> Does not look as it was not found: https://artifacts.dev.testing-farm.io/78e9d3cd-7980-472a-920e-1e7b4d6454ae/

PackIt finds tests because it uses a GitHub repository. For downstream is used downstream repo:
https://artifacts.dev.testing-farm.io/e0b683ea-6977-4e6f-b034-e6daf66ac1bd/

Reply:
Got it.

Reply:
I check how PackIt uses tmt etc. PackIt discovers tests from upstream. I will change it in another PR.